File: modules/code-py-parser/sandbox/samples/sample.py
"""示例 Python 模块：演示函数与类切块。"""

import math


def hello(name: str) -> str:
    """打招呼。"""
    return f"hello {name}"


class Greeter:
    """问候器。"""

    def greet(self) -> str:
        return hello("world")


# 尾部说明：用于检索的注释块
# 第二行注释

File: modules/code-php-parser/sandbox/samples/sample.php
<?php
/**
 * 示例 PHP 文件
 * 用于切块测试
 */

// 工具函数
function hello($name) {
    return "hello " . $name;
}

class Greeter {
    public function greet() {
        return hello("world");
    }
}
